lua/terrortown/entities/items/item_base/shared.lua

```diff
@@ -37,6 +37,9 @@ if CLIENT then
 	--     desc = "Text."
 	-- }
 
+	-- A short description for the sidebar
+	ITEM.sidebarDescription = nil
```

Comment:
I think we also need this maybe on the STATUS struct for documentation purposes. Also please document what this can be set to (string or table and its structure)

Reply:
the status struct is horrible (blame on me) and documentation is not really possible there. We could use the table thingy that we discussed in the other PR though. Because it only has a "data" table

Reply:
yes that would be good here

Reply:
will do that once @Histalek shows me how to do it

Reply:
i don't follow. i think i'm missing context?

as for the documentation i think an inline comment would be fine for now. Proper documentation will be possible once we adopt the lua language server annotations i think, but that's a bit in the future

Reply:
> i don't follow. i think i'm missing context?
> 
> as for the documentation i think an inline comment would be fine for now. Proper documentation will be possible once we adopt the lua language server annotations i think, but that's a bit in the future

In another PR we discussed something like --@param table data {param1, param2} etc. I don't know the correct syntax though. I think @EntranceJew brought it up first

Reply:
that would probably be this [1], examples for tables are in the table ( :D ) at the end of that paragraph.

[1] https://luals.github.io/wiki/annotations/#documenting-types

Reply:
> that would probably be this [1], examples for tables are in the table ( :D ) at the end of that paragraph.
> 
> [1] https://luals.github.io/wiki/annotations/#documenting-types

done